### ISSUE TITLE
Blurb on requests

### ISSUE THREAD

**Issue by the8472** [arvidn/bittorrent-multi-have#1]
> 1.  The number of request messages that would need to be sent per byte of payload byte received.

That doesn't seem true. Am I missing something? You always send requests for 16KiB-chunks, no matter the piece size.

In fact, the request usage should go down a tiny bit since endgame mode will be shorter.

**Comment by arvidn** [comment]
yeah, it doesn't look right. I'm trying to understand what I meant by that. I think that maybe I meant "have messages" instead of "request messages", and was trying to distinguish between the "cost" of not coalescing the reports of what data one has. What makes me thing that is that I seem to deliberately not call out (4) as something this proposal is trying to address. But even changing it to have messages, it doesn't make much sense, nor is distinguished from (3).

I intend to write up the HAVE coalescing we talked about in a separate BEP.
